### ISSUE TITLE
Win64 avbin load fails on avbin 1.2alpha

### ISSUE THREAD

**Issue by jaybo** [psychopy/psychopy#637]
I don't know the background of the recent merge for "Psychopy OSX 64-bit compatibility added" https://github.com/psychopy/psychopy/pull/561 but it seems to have introduced a bug on Win64 and the current avbin distribution.  

The current avbin release on Win64 from http://avbin.github.io/AVbin/Download.html reports itself as "1.2alpha" instead of just "1.2".  From window.py, line 22:

```
 if pyglet.version < "1.2":  # this is False for '1.2alpha'  (correct?)
```

As a result, haveAvbin is never defined and line 66 results in an exception:

```
if sys.platform == 'win32' and not haveAvbin:
```

If I disable the pyglet.version < '1.2' test, and allow window.py lines 22 through 44 to execute, then it all works again.
In summary, I'm not sure where the bug is, but avbin isn't loading on Win64 with the dev bits of psychopy.
